### PR TITLE
[doc] Fix typo x2 in DeploymentChain

### PR DESCRIPTION
**Pull request by t-kikuc**
**What this PR does / why we need it**:

as title

**Which issue(s) this PR fixes**:

**Does this PR introduce a user-facing change?**: no

- **How are users affected by this change**: no
- **Is this breaking change**: no
- **How to migrate (if breaking change)**: no

### CODE REVIEW

**Review by khanhtc1202 — DISMISSED** [review]
Nice catch, thanks ⚾️

File: docs/content/en/docs-dev/user-guide/managing-application/deployment-chain.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: >
   Specific guide for configuring chain of deployments.
 ---
 
-For users who want to use PipeCD to build a complex deployment flow, which contains multiple applications across multiple application kinds and roll out them to multiple clusters gradually or promoting across environments, this guideline will show you how to use PipeCD to archive that requirement.
```
Comment (ffjlabo) [inline docs/content/en/docs-dev/user-guide/managing-application/deployment-chain.md:9]: Thank you :) There might be the same typo in the other version's docs.
Could you check them?

Comment (t-kikuc) [reply docs/content/en/docs-dev/user-guide/managing-application/deployment-chain.md:9]: @ffjlabo Thanks, I fixed v0.39.x - v0.46.x too!

**Review by khanhtc1202 — APPROVED** [review]
LGTM

**Review by ffjlabo — APPROVED** [review]
🚀

**Pull request merged**